packages/react-native-editor/src/test/index.test.js

```diff
@@ -176,9 +176,19 @@ describe( 'Register Gutenberg', () => {
 		expect( hookCallOrder ).toBeGreaterThan( onRenderEditorCallOrder );
 	} );
 
-	it( 'initializes the editor', () => {
-		const { getByTestId } = initGutenberg();
-		const blockList = waitFor( () => getByTestId( 'block-list-wrapper' ) );
+	it( 'initializes the editor', async () => {
+		const MockEditor = () => <Text>Mock Editor</Text>;
+		jest.mock( '../setup', () => {
+			return {
+				__esModule: true,
+				default: jest.fn( () => <MockEditor /> ),
+			};
+		} );
+
+		const screen = initGutenberg();
+		const blockList = await waitFor( () =>
```

Comment:
The missing `await` for the asynchronous query meant the test assertion succeeded as a false-positive.